Source/Atomic/Resource/Image.cpp

```diff
@@ -2034,7 +2124,12 @@ unsigned char* Image::GetImageData(Deserializer& source, int& width, int& height
 
     SharedArrayPtr<unsigned char> buffer(new unsigned char[dataSize]);
     source.Read(buffer.Get(), dataSize);
-    return stbi_load_from_memory(buffer.Get(), dataSize, &width, &height, (int*)&components, 0);
+    // #623 BEGIN TODO: Crunch expects all 4 components. This isn't ideal, but since
+    // precompressed versions should always be loaded anyway it doesn't matter.
+    // Might need to handle differently per platform.
+    components = 4;
+    return stbi_load_from_memory(buffer.Get(), dataSize, &width, &height, nullptr, components);
+    // #623 END TODO: Crunch expects all 4 components. This isn't ideal, but since

```

Comment:
It needs to only happen on Windows, and only from BeginLoad. Short of introducing the knowledge of a forced alpha layer up the chain and passing it down which I would think is worse, I don't think there's really a good alternative. The ultimate solution would be to fix crunch's expectation of 4 components.